java/storage/connectors/redis/src/main/java/feast/storage/connectors/redis/retriever/OnlineRetriever.java

```diff
@@ -55,22 +59,47 @@ public OnlineRetriever(
   public List<List<Feature>> getOnlineFeatures(
       List<Map<String, ValueProto.Value>> entityRows,
       List<ServingAPIProto.FeatureReferenceV2> featureReferences,
-      List<String> entityNames) {
+      List<String> entityNames,
+      Map<String, List<String>> entityNamesPerFeatureView) {
+
+    List<String> featureViewNames =
```

Comment:
Isn't 65-72 equivalent to `Set(entityNamesPerFeatureView.values())` ?

Reply:
You probably meant `.keySet`, not `.values`, but you're correct, thanks!